CMS-Enterprise/easi-app
# [EASi-4566] add date columns

# EASI-4566

## Description

- UI for new columns (Upcoming meeting date, Most recent meeting date)
- Open requests view -> Upcoming
- Closed requests view -> Most recent

<img width="1277" alt="Screenshot 2024-09-04 at 13 04 09" src="https://github.com/user-attachments/assets/35417f62-5b7e-43be-ace1-b96c9918a1e4">
<img width="1240" alt="Screenshot 2024-09-04 at 13 04 20" src="https://github.com/user-attachments/assets/2d9711ac-b52d-434a-b5d9-13f1d0d409c6">


## How to test this change

- enable your `systemWorkspace` feature flag (i did this locally in order to view those pages)
- create a system intake and link to one of the systems
- progress that intake to a meeting stage, and set the meeting date for after "now"
- nav to system workspace for the system linked to the intake
- confirm the upcoming meeting date looks as expected

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
